.github/workflows/release-wasm.yml

```diff
@@ -0,0 +1,133 @@
+---
+name: Release @bitwarden/sdk-wasm
+run-name: Release @bitwarden/sdk-wasm ${{ inputs.release_type }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        description: "Release Options"
+        required: true
+        default: "Release"
+        type: choice
+        options:
+          - Release
+          - Redeploy
+          - Dry Run
```

Comment:
```suggestion
        options:
          - Release
          - Dry Run
```

Reply:
@dani-garcia I forgot about this one yesterday, sorry.

Reply:
No problem, I've removed it now!

Reply:
Thanks!